presto-product-tests/src/main/resources/sql-tests/testcases/window_functions/rowsSameWindow.sql

```diff
@@ -0,0 +1,10 @@
+-- database: presto; groups: window;
```

Comment:
I'd call it "identical" window as it's called in https://github.com/prestodb/presto/issues/4358